packages/rancher-sriov/generated-changes/overlay/templates/certmanagercerts.yaml

```diff
@@ -0,0 +1,43 @@
+{{- if .Values.operator.enableAdmissionController -}}
+{{- if .Values.cert_manager -}}
+{{- if not (.Capabilities.APIVersions.Has "cert-manager.io/v1") -}}
+{{- required "cert-manager is required but not found" "" -}}
+{{- end -}}
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: sriov-network-operator-selfsigned-issuer
```

Comment:
Should the naming scheme for these be based on `{{ include "sriov-network-operator.fullname" . }}`?

Reply:
I did not want to add another delta with regards to upstream. I'd suggest we wait until upstream implements this

Reply:
You are right, this is currently us supporting it. There are a couple of PRs upstream adding the certmanager stuff and they are not yet merged. Until then, we must support it ourselves. I have just looked at why it does the `{{ include "sriov-network-operator.fullname" . }}` instead of adding the string directly. I see that it is just in case there is a fullnameOverride in the values. We could add this here too but I don't see a big benefit of doing it

Reply:
If we are going to deprecate our certmanager stuff in favor of an eventual upstream change, I think it's fine to leave it as-is. Resolving this comment.